go.mod

```diff
@@ -38,7 +38,7 @@ require (
 	github.com/layer5io/meshery-operator v0.5.2
 	github.com/layer5io/meshkit v0.5.6
 	github.com/layer5io/meshsync v0.5.0
-	github.com/layer5io/nighthawk-go v0.5.0
```

Comment:
nighthawk-go v0.5.0 is newer than v1.0.3: https://github.com/layer5io/getnighthawk/releases/tag/v0.5.0, v0.5.0 contains the field changes, so changing to v1.0.3 will cause the failure. 
I guess the bot will update package to the latest version by default automatically. What's the intention that the latest version was name v0.5.0? @leecalcote

Reply:
getnighthawk is still evolving. And it only makes sense to reflect that on the version of getnighthawk.

A 0. version suggests that next releases can have breaking changes which it likely will considering the changes we are proposing around performance management with Meshery.

We can close this PR and not update this. I'm not sure if this will pop up again.

Reply:
It makes sense,  thanks!